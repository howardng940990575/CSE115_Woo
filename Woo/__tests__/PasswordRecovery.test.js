import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import Recovery from '../src/PasswordRecovery.js'

test('Password Recovery SnapShot', ()=> {
    const snap = renderer.create(<Recovery />).toJSON();
    expect(snap).toMatchSnapshot();
});
