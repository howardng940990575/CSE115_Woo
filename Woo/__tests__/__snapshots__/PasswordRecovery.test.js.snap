// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Password Recovery SnapShot 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#73BFF1",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View
    style={
      Object {
        "bottom": 0,
        "left": 0,
        "position": "absolute",
        "right": 0,
        "top": 0,
      }
    }
  >
    <Image
      blurRadius={5}
      source={
        Object {
          "process": [Function],
        }
      }
      style={
        Object {
          "flex": 1,
          "height": null,
          "width": null,
        }
      }
    />
  </View>
  <TextInput
    allowFontScaling={true}
    autoCapitalize="none"
    onChangeText={[Function]}
    placeholder="Email"
    rejectResponderTermination={true}
    style={
      Object {
        "backgroundColor": "white",
        "borderColor": "rgba(0,0,0,0.2)",
        "borderRadius": 25,
        "borderWidth": 0.5,
        "height": 35,
        "marginBottom": 20,
        "marginHorizontal": 20,
        "marginLeft": 50,
        "marginRight": 50,
        "marginVertical": 5,
        "paddingLeft": 10,
        "width": "75%",
      }
    }
    underlineColorAndroid="transparent"
    value=""
  />
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "white",
          "borderRadius": 35,
          "height": 50,
          "justifyContent": "center",
          "marginBottom": 40,
          "marginHorizontal": 20,
          "marginLeft": 50,
          "marginRight": 50,
          "marginVertical": 5,
          "shadowColor": "black",
          "shadowOffset": Object {
            "height": 2,
            "width": 2,
          },
          "shadowOpacity": 0.2,
          "width": 250,
        }
      }
    >
      <Text
        style={
          Object {
            "fontSize": 20,
            "fontWeight": "bold",
          }
        }
      >
        Reset Password
      </Text>
    </View>
  </View>
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "white",
          "borderRadius": 40,
          "height": 40,
          "justifyContent": "center",
          "marginBottom": 10,
          "shadowColor": "black",
          "shadowOffset": Object {
            "height": 2,
            "width": 2,
          },
          "shadowOpacity": 0.2,
          "width": 40,
        }
      }
    >
      <Text
        style={
          Object {
            "fontSize": 15,
          }
        }
      >
        X
      </Text>
    </View>
  </View>
</View>
`;
